views/help.php

```diff
@@ -0,0 +1,591 @@
+    );
+
+    // Check for general routes
+    if (isset($routeToHref[$referrer])) {
+        $href = $routeToHref[$referrer];
+    }
+    // If not found, be more specific
+    else {
+        if (startsWith($referrer, "/participant-search") ||
+            startsWith($referrer, "/view-participant") ||
+            startsWith($referrer, "/ps-view-participant")) {
+            $href = '#agency';
+        }
+        // Curricula
+        else if (startsWith($referrer, '/curricula/view')) {
+            $href = '#curricula';
+        } else if (startsWith($referrer, '/curricula/delete')) {
+            $href = '#curricula-delete';
+        } else if (startsWith($referrer, '/curricula/classes')) {
+            $href = '#curricula-add-class';
+        } else if (startsWith($referrer, '/curricula/edit')) {
+            $href = '#curricula-edit';
+        }
+        // Classes
+        else if (startsWith($referrer, '/classes/view') ||
+            startsWith($referrer, '/classes/edit')) {
+            $href = '#classes';
+        } else if (startsWith($referrer, '/classes/delete')) {
+            $href = '#classes-delete';
+        }
+        // User Management
+        else if (startsWith($referrer, '/account-settings/modify')) {
+            $href = '#user-manage-edit';
+        } else if (startsWith($referrer, '/account-settings')) {
+            $href = '#user-manage';
+        }
+    }
+}
+
+$figureCounter = 1;
+
+include('header.php');
+?>
+
+<div data-spy="scroll" data-target="#toc-list" data-offset="75" id="help-section" style="flex: 1; padding: 0 1rem">
+    <div style="width: 100%; max-width: 800px; margin: 0 auto">
+        <h4 id="general">General</h4>
+        <p>
+            This page provides detailed descriptions of various user functions in the PEP Manager. If after looking
+            through the help sections you still have not solved your problem, please contact IT for further information.
+        </p>
+        <h5 id="display">Display</h5>
+        <p>
+            The PEP Manager display is set up in three parts: the top bar, the side menu, and the main content section.
+        </p>
+        <ul>
+            <li>
+                <b>Top Bar</b> - This bar contains the name of the system, the current page you are on, and user
+                specific links (Logout, Account Settings, Help).
+            </li>
+            <li>
+                <b>Side Menu</b> - This menu contains the navigation links to get to every page within the PEP Manager.
+            </li>
+            <li>
+                <b>Main Content</b> - This is the largest portion of the page which will display all of the content
+                for each page.
+            </li>
+        </ul>
+
+        <?php if (hasRole(Role::User)) { ?>
+            <h4 id="agency">Agency Requests</h4>
+            <div>
+                <p>
+                    The Agency Requests page provides a search engine for the program’s participants, where a user can
```

Comment:
For consistency I think "a user" can be replaced with "you". You use "you" in the other help sections.